server/src/test/java/io/crate/integrationtests/SeqNoBasedOCCIntegrationTest.java

```diff
@@ -126,4 +126,26 @@ public void testSelectWhereSeqNoAndPTWithoutPrimaryKey() throws Exception {
                                 HttpResponseStatus.BAD_REQUEST,
                                 4000));
     }
+
+    @Test
+    public void test_update_where_id_and_seq_missing_primary_term() throws Exception {
```

Comment:
Isn't this already tested in `testUpdateUsingSeqNoRequiresPk` (`UpdatePlannerTest`) ?